lib/media/content_workarounds.js

```diff
@@ -69,6 +69,12 @@ shaka.media.ContentWorkarounds = class {
         })
         .fullBox('encv', onEncryptionMetadataBox)
         .fullBox('enca', onEncryptionMetadataBox)
+        .fullBox('hvc1', (box) => {
```

Comment:
Please, add hev1 also

Reply:
Just pushed a commit to fulfill your request.

Reply:
Thanks!